src/main/java/com/bermudalocket/lastseen/LastSeen.java

```diff
@@ -126,6 +164,19 @@ private static String longToDate(Long time) {
         return DATE_FORMAT.format(CALENDAR.getTime());
     }
 
+    // ------------------------------------------------------------------------
+    /**
+     * Turns the given timestamp into a string describing the relative date
+     * in English to the current time now.
+     *
+     * @param time the timestamp.
+     * @return a string describing the relative date.
+     */
+    private static String longToRelativeDate(Long time) {
+        CALENDAR.setTimeInMillis(time);
+        return PRETTY_TIME_FORMAT.format(CALENDAR.getTime());
```

Comment:
Can you elaborate more on this change?

Reply:
Sorry, I neglected to include a description.  I probably should have created an issue to track.  Will do so in the future.  See new PR description: https://github.com/bermudalocket/LastSeen/pull/2#issue-279588324